packages/pwa-kit-react-sdk/package.json

```diff
@@ -46,7 +46,8 @@
     "react-ssr-prepass": "^1.5.0",
     "react-uid": "^2.3.2",
     "serialize-javascript": "^6.0.2",
-    "svg-sprite-loader": "^6.0.11"
+    "svg-sprite-loader": "^6.0.11",
+    "winston": "^3.13.0"
```

Comment:
Do we need a 3PP for this?

Reply:
The 3pp [winston](https://www.npmjs.com/package/winston) is already approved.